org.eclipse.xtend.lib.gwt.test/build.gradle

```diff
@@ -5,8 +5,9 @@ plugins {
 group = 'org.eclipse.xtend'
 
 dependencies {
-	compile project(':org.eclipse.xtend.lib.gwt')
-	compile "com.google.gwt:gwt-dev:$versions.gwt"
+	api project(':org.eclipse.xtend.lib.gwt')
+	gwt project(':org.eclipse.xtend.lib.gwt')
```

Comment:
somehow with api only the resources of the project are not visible to the plugin

Reply:
Probably an outdated plugin then, might break with future versions.